keyboards/sneakbox/aliceclone/info.json

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
This is the wrong file format, needs to be <https://docs.qmk.fm/#/reference_info_json> not via.

Reply:
My apologies. Yes,makes sense to me now. Thank you. I updated info.json to reflect the proper QMK configurator metadata, including switch positions and per-switch angles, instead of via metadata.

Reply:
It's still a bit off, please remove the rotation. Configurator curently doesn't support rotated keys.
![image](https://user-images.githubusercontent.com/4781841/88473850-e42c2900-cf64-11ea-8c36-827fb4195a2a.png)

Reply:
Corrected. Thanks.

Reply:
Sorry, but if you change the LAYOUT macro, you'll need to redo this, again. :(

Reply:
Thanks for the feedback. Could you elaborate what you mean about redoing the LAYOUT macro in the configurator json file? The change I made was to undo the rotation, and alight the keys to their respective rows in a linear fashion.